examples/core/transform-feedback-instanced/app.js

```diff
@@ -176,80 +158,60 @@ const animationLoop = new AnimationLoop({
       instanceColors[i * 3 + 2] = Math.random();
     }
 
-    // -- Init VertexArrays and TransformFeedback objects
-
-    const vertexArrays = new Array(2);
-    const transformVertexArrays = new Array(2);
-    const transformFeedbacks = new Array(2);
-
     const positionBuffer = new Buffer(gl, {
       data: trianglePositions,
       size: 2,
-      type: gl.FLOAT
+      type: gl.FLOAT,
+      instanced: 0
     });
     const colorBuffer = new Buffer(gl, {
       data: instanceColors,
       size: 3,
+      type: gl.FLOAT,
+      instanced: 1
+    });
+
+    const offsetBuffer = new Buffer(gl, {
+      data: instanceOffsets,
+      size: 2,
       type: gl.FLOAT
     });
 
-    for (let va = 0; va < vertexArrays.length; ++va) {
-      const offsetBuffer = new Buffer(gl, {
-        data: instanceOffsets,
-        size: 2,
-        type: gl.FLOAT
-      });
-
-      const rotationBuffer = new Buffer(gl, {
-        data: instanceRotations,
-        size: 1,
-        type: gl.FLOAT
-      });
-
-      vertexArrays[va] = new VertexArray(gl, {
-        buffers: {
-          [COLOR_LOCATION]: {
-            buffer: colorBuffer,
-            instanced: 1
-          },
-          [OFFSET_LOCATION]: {
-            buffer: offsetBuffer,
-            instanced: 1
-          },
-          [ROTATION_LOCATION]: {
-            buffer: rotationBuffer,
-            instanced: 1
-          },
-          [POSITION_LOCATION]: {
-            buffer: positionBuffer,
-            instanced: 0
-          }
-        }
-      });
-
-      transformVertexArrays[va] = new VertexArray(gl, {
-        buffers: {
-          [OFFSET_LOCATION]: {
-            buffer: offsetBuffer,
-            instanced: 0
-          },
-          [ROTATION_LOCATION]: {
-            buffer: rotationBuffer,
-            instanced: 0
-          }
-        }
-      });
-
-      /* eslint-disable camelcase  */
-      transformFeedbacks[va] = new TransformFeedback(gl, {
-        buffers: {
-          v_rotation: rotationBuffer,
-          v_offset: offsetBuffer
-        },
-        varyingMap: modelTransform.varyingMap
-      });
-      /* eslint-enable camelcase  */
-    }
+    const rotationBuffer = new Buffer(gl, {
+      data: instanceRotations,
+      size: 1,
+      type: gl.FLOAT
+    });
+
+    /* eslint-disable camelcase  */
+    const modelRender = new Model(gl, {
+      id: 'Model-Render',
+      vs: DRAW_VS,
+      fs: DRAW_FS,
+      drawMode: gl.TRIANGLE_FAN,
+      vertexCount: 3,
```

Comment:
It seems to me that Model could autodeduce vertexCount, isInstanced and instanceCount from buffers. That would be a separate PR but a quite nice feature I think.